configs/configupgrade/test-fixtures/valid/invalid-resource-name/want/resource.tf

```diff
@@ -0,0 +1,8 @@
+# TF-UPGRADE-TODO: In Terraform v0.11 and earlier, it was possible to begin a
```

Comment:
unrelated, I was just annoyed by how long it took me to understand what was happening in a test _I wrote_.